charts/kubeclarity/templates/secret.yaml

```diff
@@ -3,7 +3,25 @@
   {{- $secretName = index .Values "kubeclarity-postgresql-external" "auth" "existingSecret" -}}
 {{- end }}
 {{- $secretKey := index .Values "kubeclarity-postgresql-secret" "secretKey" -}}
+{{- $usernameKey := index .Values "kubeclarity-postgresql-secret" "usernameKey" -}}
+{{- $hostKey := index .Values "kubeclarity-postgresql-secret" "hostKey" -}}
+{{- $portKey := index .Values "kubeclarity-postgresql-secret" "portKey" -}}
+{{- $databaseKey := index .Values "kubeclarity-postgresql-secret" "databaseKey" -}}
+{{- $sslmodeKey := index .Values "kubeclarity-postgresql-secret" "sslmodeKey" -}}
 {{- $postgresPassword := .Values.global.databasePassword | b64enc -}}
+{{- $dbHost := print (include "kubeclarity.name" .) "-postgresql" | b64enc -}}
+{{- $dbPort := index .Values "kubeclarity-postgresql" "service" "ports" "postgresql" | toString | b64enc -}}
+{{- $dbUser := index .Values "kubeclarity-postgresql" "auth" "username" | b64enc -}}
+{{- $dbName := index .Values "kubeclarity-postgresql" "auth" "database" | b64enc -}}
+{{- $dbSSLMode := index .Values "kubeclarity-postgresql" "auth" "sslMode" | b64enc -}}
+{{ if index .Values "kubeclarity-postgresql-external" "enabled" }}
+  {{- $secretName = index .Values "kubeclarity-postgresql-external" "auth" "existingSecret" | b64enc -}}
```

Comment:
This variable was already handled by L1-4 so I think we should clean that up now we're handling it all here.

Reply:
Thanks for the hint, unfortunately I had overlooked that.